docker-compose.yml

```diff
@@ -5,32 +5,35 @@ volumes:
 services:
   tesla_ble_mqtt:
     container_name: tesla_ble_mqtt
-    image: "tesla-local-control/tesla_ble_mqtt:latest"
+    image: "iainbullock/tesla_ble_mqtt:latest"
 
-
-    # Configuration Settings have been moved to the "env" file
+    # Configuration Settings have been moved to the "stack.env" file
     env_file:
-      - env
+      - stack.env
 
+    # Needed to open the Bluetooth dongle to send commands to the car
     network_mode: host
-
-    # TODO:  Needed to open the Bluetooth dongle to send commands to the car
     privileged: true
-    # TODO: Needed to open the Bluetooth device
+
+    # TODO: Needed to open the Bluetooth device. 
+    # Reply: I think it is needed to get a container shell which is a useful diagnostic
```

Comment:
Indeed - very possible. I will test this when I have the time, just so I understand :)